content/first-order-logic/syntax-and-semantics/subformulas.tex

```diff
@@ -87,4 +87,25 @@
 !!{subformula}s of the ``earlier'' !!{formula}s $!B$ and $!C$.
 \end{explain}
 
+\begin{prop}
+\ollabel{prop:subfrm-trans}
+Suppose $!B$ is a subformula of $!A$ and $!C$ is a subformula of $!B$.
+Then $!C$ is a subformula of $!A$. In other words, the subformula
+relation is transitive.
+\end{prop}
+
+\begin{prob}
+Prove \olref{prop:subfrm-trans}.
```

Comment:
Can you put full references here (\olref[..][..][..]{..})?

Reply:
(and below)

Reply:
Done in [f0e48d7](https://github.com/OpenLogicProject/OpenLogic/pull/317/commits/f0e48d7b69fd4f138fac1077ae74a5a5d3590fca).